demo/dh-prod.yaml

```diff
@@ -0,0 +1,409 @@
+
```

Comment:
currently, dh-prod and dh-localhost are almost exactly the same.  Really, the prod version is going to be broken up into a helm chart with N yaml files instead of all mixed together like this.

Reply:
Review my comments on both files, in case I missed a comment.